aodncore/pipeline/steps/harvest.py

```diff
@@ -435,7 +436,13 @@ def run(self, pipeline_files):
         self.pipeline_files = pipeline_files
         objs = self.params.get('db_objects')
         if objs:
-            runsheet = (x for x in map(self.build_runsheet, objs) if x)
+            runsheet = [x for x in map(self.build_runsheet, objs) if x]
```

Comment:
changed _runsheet_ from generator to list

Reply:
:+1: Thanks. That'll do it!